KodiController/test/controller.js

```diff
@@ -33,6 +41,21 @@ describe("Controller", () => {
 		});
 	});
 
+	describe("getVolume", () => {
+		it("should return the volume of the player", (done) => {
+			let c = new Controller();
+
+			SetNockVolume();
+
+			c.getVolume((err, volume) => {
+				if (err || !volume) done(err);
+				console.log("THE VOLUME IS", err);
```

Comment:
should be `console.log("THE VOLUME IS", volume);`,  or just delete that line

Reply:
Good catch, removed